### ISSUE TITLE
helm-chart-aws-ebs-csi-driver-2.6.0 ebs-csi-node docker image v1.6.0 is not available.

### ISSUE THREAD

**Issue by sperahd** [kubernetes-sigs/aws-ebs-csi-driver#1143]
/kind bug
/priority critical-urgent

helm-chart-aws-ebs-csi-driver-2.6.0 has appVersion:1.6.0 which is defaulting as the docker image version for ebs-csi-node. The corresponding docker image doesn't seem to be present in the repository.

```
Annotations:        kubernetes.io/psp: eks.privileged
Status:             Pending
IP:                 172.31.82.27
Controlled By:      DaemonSet/ebs-csi-node
Containers:
  ebs-plugin:
    Container ID:  
    Image:         k8s.gcr.io/provider-aws/aws-ebs-csi-driver:v1.6.0
    Image ID:      
    Port:          9808/TCP
    Host Port:     9808/TCP
    Args:
      node
      --endpoint=$(CSI_ENDPOINT)
      --logtostderr
      --v=2
    State:          Waiting
      Reason:       ImagePullBackOff
```

The docker image v1.6.0 should have been present in the repository before the helm-chart release was made.

Install https://github.com/kubernetes-sigs/aws-ebs-csi-driver/releases/tag/helm-chart-aws-ebs-csi-driver-2.6.0 helm chart, newly provisioned instances are having docker images with v1.6.0 which leads to ImagePullBackOff

**Environment**
- Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.0", GitCommit:"af46c47ce925f4c4ad5cc8d1fca46c7b77d13b38", GitTreeState:"clean", BuildDate:"2020-12-08T17:59:43Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"20+", GitVersion:"v1.20.7-eks-d88609", GitCommit:"d886092805d5cc3a47ed5cf0c43de38ce442dfcb", GitTreeState:"clean", BuildDate:"2021-07-31T00:29:12Z", GoVersion:"go1.15.12", Compiler:"gc", Platform:"linux/amd64"}

- Driver version:

**Comment by stevehipwell** [comment]
Exactly the same here.

**Comment by jotasixto** [comment]
I have the same problem :cry:

**Comment by B3arp** [comment]
Same issue.

**Comment by Setheck** [comment]
Same thing here too. I followed the aws guide and set the image.repository in the helm char to `602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-ebs-csi-driver` as per https://docs.aws.amazon.com/eks/latest/userguide/add-ons-images.html
It was working fine until what looks like 12 hours ago.
Now it looks like `602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-ebs-csi-driver:v1.6.0` is no longer in the registry...

**Comment by sperahd** [comment]
@Setheck Looks like the release was never tagged and hence the image was never pushed. We cannot see v1.6.0 tag on the releases page https://github.com/kubernetes-sigs/aws-ebs-csi-driver/releases

**Comment by gtxu** [comment]
The newest version of aws-ebs-csi-driver is v1.5.0, and upcoming v1.5.1 to fix a security issue, there's no v1.6.0 currently.

**Comment by sperahd** [comment]
> The latest version of aws-ebs-csi-driver is v1.5.0, and upcoming v1.5.1 to fix a security issue, there's no v1.6.0 currently.

@gtxu then shouldn't the chart.yaml AppVersion be v1.5.0 instead of v1.6.0?

**Comment by gtxu** [comment]
yeah, a PR is ongoing fix this #1145
